Gemfile

```diff
@@ -99,6 +99,8 @@ gem 'griddler-sparkpost'
 
 gem 'rails-timeago'
 
+gem 'devise_invitable', '~> 1.5.2'
+
```

Comment:
@shaktik1989 I discovered this needs `~> 1.6` to work with Rails 4.2.7 and Rails 5